lib/ButtonNew/ButtonPrimary.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cx from 'classnames';
+import { ButtonBase } from './ButtonBase';
+import { propTypes, defaultProps, sizes } from './common';
+
+function ButtonPrimary({
+  children,
+  className,
+  disabled,
+  connectedRight,
+  danger,
+  ...rest
+}) {
+  const classes = cx('text-white ', className, {
+    'bg-blue-primary': !danger,
+    'bg-red-primary': danger,
+    'bg-neutral-90 text-neutral-primary': disabled,
+    'border-blue-40': connectedRight && !disabled,
+    'border-neutral-80': connectedRight && disabled,
+    'hover:bg-blue-60 hover:shadow-blue active:bg-blue-40 active:shadow-blue-inset':
```

Comment:
💚same comment as above, and if blue & red were separate then we'd lose a couple of these conditional styles 👀